app/models/git_hub_user.rb

```diff
@@ -15,8 +15,12 @@ def login
   # Public
   #
   def admin_organization_memberships
+    no_cache_headers = { 'Cache-Control' => 'no-cache, no-store' }
```

Comment:
That sounds right, but I don't think the useragent can send the `Cache-Control` header. That's a header typically sent by the server to instruct the *useragent* not to cache the response. I'm not sure that the code change here does anything, but I could be totally wrong.